src/background/index.ts

```
@@ -671,6 +682,84 @@ function handleGetTabsPort(
+): void {
+  try {
+    const config = BrowserOSProvidersConfigSchema.parse(payload)
+    const browserOS = (chrome as any)?.browserOS as { setPref?: (name: string, value: any, pageId?: string, cb?: (ok: boolean) => void) => void } | undefined
+    if (browserOS?.setPref) {
+      browserOS.setPref(
+        BROWSEROS_PREFERENCE_KEYS.PROVIDERS,
+        JSON.stringify(config),
+        undefined,
+        (success?: boolean) => {
+          if (success) {
+            try { langChainProvider.clearCache() } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)